src/stylesheets/elements/form-controls/_combo-box.scss

```diff
@@ -14,6 +37,13 @@
 
   .usa-combo-box__clear-input {
     display: block;
+
+    @media (forced-colors: active) {
+      @include add-color-icon($close);
+      height: 1.5rem;
+      width: auto;
+      top: 0.5rem;
+    }
```

Comment:
# Combo-box

This update keeps the background svg in standard views but places the icon mask in `forced-colors` is set to active.

## Currently 

Standard
![image](https://user-images.githubusercontent.com/25211650/156629765-3e1962cb-3739-4227-a0ee-6acfbce00c4c.png)

High Contrast mode
![image](https://user-images.githubusercontent.com/25211650/156636493-52910194-9806-44ad-8961-28846eb54220.png)

## After Fix

![image](https://user-images.githubusercontent.com/25211650/156635974-0540465a-5ee4-47f1-8a90-ca2bfbaa8d0b.png)